src/main/scala/net/artsy/atomic/AtomicEventStore.scala

```diff
@@ -203,32 +205,25 @@ abstract class AtomicEventStore[EventType <: Serializable: Scoped: ClassTag, Val
   class Receptionist(
     logProps: String => Props
   ) extends Actor {
-    var logs = Map.empty[String, ActorRef]
+    val logs = scala.collection.mutable.Map.empty[String, ActorRef]
 
-    def liveLogForScope(scope: String): ActorRef = {
-      val (newLogs, targetLog) = logs.get(scope) match {
-        case Some(foundLog) => (logs, foundLog)
-        case None =>
-          // Recreate the log, which will recall all preexisting events
-          val materializedLog = context.actorOf(logProps(scope), scope)
+    def liveLogForScope(scope: String): ActorRef = logs.getOrElseUpdate(scope, createLog(scope))
```

Comment:
Yeah, I think chaining wouldn't necessarily be most readable, but with an intermediate variable, it would be.

My issue with small functions is the mental call stack gets a little deeper, so I think there is some cost there, if the function isn't reused.

But that's a minor opinion. Feel free to merge as-is!